.github/CONTRIBUTING.txt

```diff
@@ -13,6 +13,8 @@ Here's the long and short of it:
 
       git clone https://github.com/your-username/scikit-image.git
 
+   * cd scikit-image
```

Comment:
How about putting something like:
```
* Change the directory::

    cd scikit-image
```
This way it will be visualized as a source code block.